client/my-sites/customer-home/cards/tasks/domain-upsell/style.scss

```diff
@@ -0,0 +1,115 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.task {
+	@media (max-width: $break-large) {
+		flex-direction: column;
+	}
```

Comment:
I think this may need to be nested under `&.task__domain-upsell` otherwise the image icon on the other banners goes a little wonky.
 
<img src="https://user-images.githubusercontent.com/140841/231221814-185f47c4-ea80-4e04-a991-377d87cf8677.png" width="300" />

Reply:
You're right, I will do it!